src/code/z_view.c

```diff
@@ -255,17 +263,23 @@ s32 View_StepDistortion(View* view, Mtx* projectionMtx) {
     return true;
 }
 
-void func_800AAA50(View* view, s32 arg1) {
-    arg1 = (view->flags & arg1) | (arg1 >> 4);
+/**
+ * Apply view to POLY_OPA_DISP, POLY_XLU_DISP, OVERLAY_DISP
+ */
+void View_Apply(View* view, s32 mask) {
```

Comment:
only `View_ApplyOrtho` appends to overlay_disp, and even then only sets the viewport. so I guess overlay_disp may be omitted here.

It doesn't matter for overlay_disp anyway since it's meant to be used to draw 2d rectangles (*) in screen coordinates, bypassing the viewport (and projection) entirely if I understand it right

(*) or so I thought, I looked at what uses `View_ApplyOrthoToOverlay` since it wouldn't make sense to set a viewport on overlay_disp then, and actually some interface/pause menu code does draw "normal" geometry with vertices instead of 2d rectangles to overlay_disp. So now idk

Reply:
drawing something to overlay doesnt necessarily mean its 2d. The A button on the interface for example is 3d, but is drawn to overlay. So im not sure this differentiation is sufficient

Reply:
Okay, I suppose there's an implied "as necessary" in this comment. These `xxx_DISP`s, as a set, are the target of the operation. I'm not trying to say that it always touches all of them.

I don't think it's necessary for this comment to detail exactly what happens when; you can read the code for that.

Reply:
I do feel like it matters that overlay_disp is not touched at all in the perspective case
```suggestion
 * Apply view to POLY_OPA_DISP, POLY_XLU_DISP (and OVERLAY_DISP if ortho)
 */
void View_Apply(View* view, s32 mask) {
```